.changeset/five-forks-thank.md

```diff
@@ -0,0 +1,6 @@
+---
+'@cypherock/cysync-core': patch
+'@cypherock/cysync-ui': patch
```

Comment:
Add `@cypherock/cysync-desktop` to the changeset as well.

Reply:
Oh I forgot